### ISSUE TITLE
Proactively show that h is not installed

### ISSUE THREAD

**Issue by danbst** [zimbatm/h#4]
I was desperately running `h NixOS` without understanding why doesn't it work. It turned out I have to run `h` without args to get more information:
```
[danbst@station:~/code]$ h NixOS
/home/danbst/code
Usage: h (<name> | <repo>/<name> | <url>) [git opts]

[danbst@station:~/code]$ h
h is not installed

/home/danbst/code
Usage: eval "$(h --setup [code-root])"
```

**Comment by zimbatm** [comment]
Good idea

**Comment by zimbatm** [comment]
released v1.0.1 with the fix

**Comment by zimbatm** [comment]
`up` should probably also receive the same treatment
